### PR TITLE
Snapshot

### PR DESCRIPTION
**Pull request by ashleyschuett**
@nicktate @normanjoyner

### CODE REVIEW

File: handlers/v1/backup.js [hunk 1]
```diff
@@ -23,22 +23,37 @@ module.exports = {
...
+                    const volume_to_backup = _.find(volumes, (volume_without_host) => {
+                        return volume_without_host.container === volume.container;
+                    });
+
+                    if(!volume_to_backup) {
+                        return;
+                    }
+
+                    if(nodes[container.host]) {
+                        core.loggers['containership-cloud'].log('verbose', `Requesting volume backup for ${volumes.host} in container ${container.id}`);
```
Comment (nicktate) [inline handlers/v1/backup.js]: i think you want `${volume_to_backup.host}` in the log

File: handlers/v1/backup.js [hunk 2]
```diff
@@ -23,22 +23,37 @@ module.exports = {
...
+                        core.cluster.legiond.send({
+                            event: 'containership-cloud.backup',
+                            data: {
+                                path: volumes.host,
```
Comment (nicktate) [inline handlers/v1/backup.js]: should this be `volume_to_backup.host`

Comment (ashleyschuett) [inline handlers/v1/backup.js]: volume_to_backup doesn't have host in it. if you look at what volumes is, is all the volumes with host of undefined. So then I have to compare the applications volumes container to the volume in the container. Then the containers volume will have a host. If that makes sense?  

Same for your comment above.

**Comment by ashleyschuett** [comment]
@nicktate let me know if my comment doesn't make sense and I'll show you what I mean.

**Comment by nicktate** [comment]
👍  looks good to me @normanjoyner

**Pull request merged**